tests/Makefile

```diff
@@ -193,6 +193,8 @@ $(TOP)/tools/mtouch/SdkVersions.cs: $(TOP)/tools/common/SdkVersions.cs.in
 	@touch $@
 
 xharness/xharness.exe: $(wildcard xharness/*.cs) xharness/xharness.csproj $(TOP)/tools/mtouch/SdkVersions.cs test.config test-system.config .stamp-src-project-files
+	make -C $(TOP)/external/mono/mcs/class -i do-test PROFILE=MONOTOUCH
+	make -C $(TOP)/external/mono/mcs/class -i do-xunit-test PROFILE=MONOTOUCH
 	$(Q_GEN) $(SYSTEM_MSBUILD) $(MSBUILD_VERBOSITY_QUIET) /r xharness/xharness.csproj
```

Comment:
@rolfbjarne not sure about this. But we need those to be compiled so that they are consumed by the new bcl tests.